.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,3 +1,5 @@
+<!-- markdownlint-disable MD041 -->
```

Comment:
Supports the accessibility change in our GitHub things which should be an H4 for screen-readers. FMI - #788.